### DISCUSSION TITLE
shared Excel file for organization

### DISCUSSION THREAD

**Discussion by BWhannou** [ML-for-B-E/.github#4] [Announcements]
@ML-for-B-E/ml 
here you will find the Excel file we are using for organization purposes:
https://docs.google.com/spreadsheets/d/1AOx12J9GmASa02Xztr7iaI-rxc2h3ToLkP-sPInRov4/edit?usp=sharing

**Comment by akouete-kpakpo** [comment]
great! Thanks @BWhannou
